src/chat/info_save.py

```
@@ -0,0 +1,108 @@
+import mimetypes
```

Ruff (0.12.2)

`mimetypes` imported but unused

Remove unused import: `mimetypes`

(F401)